README.md

```diff
@@ -10,22 +10,59 @@ Jupyter notebooks hosted on `stitches` use functionality that is contained withi
 
 **NOTE:**  Ensure you are using Python >= 3.6.  Calling `python` may use a different instance.  Some users may need to use `python3` or the like instead.
 
-`stitches`  users and developers agree to must agree to our community guidelines outlines in our [contirbutors guidelines](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md). Open an issue to ask for help or report an issue ([how to open a github issue](https://docs.github.com/en/enterprise-server@3.1/issues/tracking-your-work-with-issues/creating-an-issue)). 
+`stitches`  users and developers agree to must agree to our community guidelines outlines in our 
+[contirbutors guidelines](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md). 
+Open an issue to ask for help or report an issue ([how to open a github issue](https://docs.github.com/en/enterprise-server@3.1/issues/tracking-your-work-with-issues/creating-an-issue)). 
 
 #### For developers
 Clone the repository and then install the package in development mode using from inside the package's root directory:
 ```bash
 python setup.py develop
 ```
 
+Before `stitches` can be used, the internal package data must be installed or
+generated.
+
+To install pre-built run the following:
+```bash
+import stitches
+stitches.install_package_data()
+```
+
+For users that would like to generate the run the following: 
```

Comment:
Dropped a few words:

For users that would like to generate the package data directly, run the following: